frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -97,13 +99,19 @@ export class Wizard {
   }
 
   async selectWorkloadProfile(workloadProfile: string) {
-    await selectOptionByText(view.workloadProfileSelect, workloadProfile);
+    await selectItemFromDropdown(view.workloadProfileSelect, view.dropDownItem(workloadProfile));
   }
 
-  async selectProvisionSource(provisionOptions) {
-    await selectOptionByText(view.provisionSourceSelect, provisionOptions.method);
-    if (Object.prototype.hasOwnProperty.call(provisionOptions, 'source')) {
-      await fillInput(view.provisionSourceInputs[provisionOptions.method], provisionOptions.source);
+  async selectProvisionSource(provisionSource: ProvisionSource) {
+    await selectItemFromDropdown(
+      view.provisionSourceSelect,
+      view.dropDownItemTitle(provisionSource.getDescription()),
+    );
+    if (provisionSource.getSource()) {
+      await fillInput(
+        view.provisionSourceInputs[provisionSource.getValue()],
```

Comment:
please use the enum also in provisionSourceInputs

Reply:
how would you suggest to use it? I used the values as keys. how else  would you do it ?

Reply:
`[ ProvisionSource.Container.getValue()]` like this? Or you can move the inputs to the provisionSource enum but please beware circular deps.